packages/server-wallet/e2e-testing/test-data/roles.json

```diff
@@ -17,6 +18,7 @@
     "chainId": 9001,
     "artifactFile": "temp/contract_artifacts.json",
     "privateKey": "0xb3ab7b031311fe1764b657a6ae7133f19bac97acd1d7edca9409daa35892e727",
-    "chainPrivateKey": "0x2030b463177db2da82908ef90fa55ddfcef56e8183caf60db464bc398e736e6f"
+    "chainPrivateKey": "0x2030b463177db2da82908ef90fa55ddfcef56e8183caf60db464bc398e736e6f",
+    "destination":"0xBBDDFb5DADcf0E710D898DB8da7861aA8F635B9b"
```

Comment:
Using a unique destination (which starts with 0 ETH) makes performing the sanity checks much easier.